engines/director/lingo/lingo-code.cpp

```diff
@@ -85,6 +86,8 @@ static struct FuncDescr {
 	{ Lingo::c_starts,		"c_starts",		"" },
 	{ Lingo::c_intersects,	"c_intersects",	"" },
 	{ Lingo::c_within,		"c_within",		"" },
+	{ Lingo::c_field,       "c_field",      "" },
+	{ Lingo::c_of,		    "c_of",		    "" },	// D9
```

Comment:
D9, as Director 9?

Reply:
I think that was a paste mistake, removed